lib/linguist/languages.yml

```diff
@@ -4995,6 +4994,15 @@ Ruby:
   - Vagrantfile
   - buildfile
   language_id: 326
+Ruby Gemfile.lock:
```

Comment:
There's no other type of `Gemfile.lock` in existence other than Ruby's, so I'd rename this to

```suggestion
Gemfile.lock:
```

Reply:
👍  done